code/solutions/architecting-templates/application-blue-green.template

```diff
@@ -0,0 +1,275 @@
+AWSTemplateFormatVersion: "2010-09-09"
+
+Description: Sample template describing an example application deployment. (uksb-1q9p31idr)
+
+Parameters:
+  AppNameTagValue:
+    Description: 'Please specify a value; minimum length: 3, maximum: 50.'
```

Comment:
Should we provide description what this parameter is e.g. `The application name to tag your infrastructure with.`

Reply:
Yes, let me update tag-related parameter descriptions accordingly, thank you!